client/components/relatedproducts/related-product-card.jsx

```diff
@@ -2,17 +2,17 @@ import React, {useState, useEffect} from 'react';
 import {StarFill} from 'react-bootstrap-icons';
 import Modal from 'react-modal';
 import ModalDetails from './modalDetails.jsx';
-import Rating from '../Overview/Rating.jsx';
 import api from '../../../api.js';
 import {StaticRating} from '../../starRating.jsx';
+import regeneratorRuntime from 'regenerator-runtime';
```

Comment:
Is this related to testing?

Reply:
@koalarick yes, it's a work-around that I found because I was initially having async errors when I was running Jest.